concurrent/actor/ActorPeriodicTask.java

```diff
@@ -43,21 +39,14 @@ public ActorPeriodicTask(Actor<STATE> actor, int timeoutMs, int variationMs, Con
         timer = null;
     }
 
-    public void restart() {
+    public void start() {
```

Comment:
from a quick chat with ganesh, this would make the most sense if called `reschedule`